lib/Sema/CSSimplify.cpp

```diff
@@ -2972,6 +2972,11 @@ performMemberLookup(ConstraintKind constraintKind, DeclName memberName,
       
     } else {
       if (!hasStaticMembers) {
+        if (cand->isImplicit() && isa<ConstructorDecl>(cand)) {
```

Comment:
I'm with Slava. I think this would incorrectly silence an inherited *required* initializer that has some other problem.